diff -Naur media-driver-intel-media-24.2.2.old/media_driver/cmake/linux/media_feature_flags_linux.cmake media-driver-intel-media-24.2.2/media_driver/cmake/linux/media_feature_flags_linux.cmake
--- media-driver-intel-media-24.2.2.old/media_driver/cmake/linux/media_feature_flags_linux.cmake	2024-05-23 14:14:33.328603130 +0200
+++ media-driver-intel-media-24.2.2/media_driver/cmake/linux/media_feature_flags_linux.cmake	2024-05-23 14:16:21.132866772 +0200
@@ -182,6 +182,9 @@
 
 if(NOT ENABLE_NONFREE_KERNELS)
     add_definitions(-D_FULL_OPEN_SOURCE)
+    add_definitions(-DBUILD_DETAILS="Free Kernel Build")
+else()
+    add_definitions(-DBUILD_DETAILS="Full Feature Build")
 endif()
 
 if(ENABLE_XE_KMD)
diff -Naur media-driver-intel-media-24.2.2.old/media_driver/linux/common/ddi/media_libva.h media-driver-intel-media-24.2.2/media_driver/linux/common/ddi/media_libva.h
--- media-driver-intel-media-24.2.2.old/media_driver/linux/common/ddi/media_libva.h	2024-05-23 14:14:33.332603143 +0200
+++ media-driver-intel-media-24.2.2/media_driver/linux/common/ddi/media_libva.h	2024-05-23 14:16:52.976946309 +0200
@@ -70,7 +70,7 @@
 #endif
 #define DDI_CODEC_GEN_MAX_ATTRIBS_TYPE             4    //VAConfigAttribRTFormat,    VAConfigAttribRateControl,    VAConfigAttribDecSliceMode,    VAConfigAttribEncPackedHeaders
 
-#define DDI_CODEC_GEN_STR_VENDOR                   "Intel iHD driver for Intel(R) Gen Graphics - " MEDIA_VERSION " (" MEDIA_VERSION_DETAILS ")"
+#define DDI_CODEC_GEN_STR_VENDOR                   "Intel iHD driver for Intel(R) Gen Graphics - " MEDIA_VERSION " (" MEDIA_VERSION_DETAILS BUILD_DETAILS ")"
 
 #define DDI_CODEC_GET_VTABLE(ctx)                  (ctx->vtable)
 #define DDI_CODEC_GET_VTABLE_VPP(ctx)              (ctx->vtable_vpp)
