diff -Naur media-driver-intel-media-26.1.6.old/CMakeLists.txt media-driver-intel-media-26.1.6/CMakeLists.txt
--- media-driver-intel-media-26.1.6.old/CMakeLists.txt	2026-04-24 10:56:33.883064144 +0200
+++ media-driver-intel-media-26.1.6/CMakeLists.txt	2026-04-24 10:56:54.284063606 +0200
@@ -18,7 +18,7 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.5...3.10)
 project(IntelMediaDriver)
 set(MEDIA_VERSION "26.1.6${MEDIA_VERSION_EXTRA}" CACHE STRING "" FORCE)
 
diff -Naur media-driver-intel-media-26.1.6.old/cmrtlib/CMakeLists.txt media-driver-intel-media-26.1.6/cmrtlib/CMakeLists.txt
--- media-driver-intel-media-26.1.6.old/cmrtlib/CMakeLists.txt	2026-04-24 10:56:33.887494483 +0200
+++ media-driver-intel-media-26.1.6/cmrtlib/CMakeLists.txt	2026-04-24 10:56:54.284324788 +0200
@@ -19,7 +19,7 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 
 set(BUILD_ALL $ENV{BUILD_ALL})
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 2.8...3.10)
 project(CM_RT)
 add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/linux)
 
diff -Naur media-driver-intel-media-26.1.6.old/cmrtlib/linux/CMakeLists.txt media-driver-intel-media-26.1.6/cmrtlib/linux/CMakeLists.txt
--- media-driver-intel-media-26.1.6.old/cmrtlib/linux/CMakeLists.txt	2026-04-24 10:56:33.892741992 +0200
+++ media-driver-intel-media-26.1.6/cmrtlib/linux/CMakeLists.txt	2026-04-24 10:56:54.284490320 +0200
@@ -18,7 +18,7 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 2.8...3.10)
 include (${CMAKE_CURRENT_LIST_DIR}/cmrt_utils.cmake)
 
 
diff -Naur media-driver-intel-media-26.1.6.old/os_release_info.cmake media-driver-intel-media-26.1.6/os_release_info.cmake
--- media-driver-intel-media-26.1.6.old/os_release_info.cmake	2026-04-24 10:56:34.761691463 +0200
+++ media-driver-intel-media-26.1.6/os_release_info.cmake	2026-04-24 10:56:54.284755330 +0200
@@ -29,7 +29,7 @@
 # of the local cmake environment.
 
 # Set cmake policies for at least this level:
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 2.8.12...3.10)
 
 
 # Function get_os_release_info - Determine and return OS name and version
